.gitignore

```diff
@@ -5,6 +5,6 @@ node_modules
 # environment variables
 .env
 .env.*
-!.env.example
+!.env.sample
```

Comment:
why is this required if the line above has `.env.*`?

Reply:
`!` prefix negates the pattern so it makes sure that `.env.sample` can be included